ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.ts

```diff
@@ -193,13 +200,17 @@ export class TissueComponent {
           if (err._body === Auth.AUTHENTICATION_ERROR) {
             this.auth.doLogout();
           }
+          result= false;
         },
       });
     }
+    return result;
   }
 
   deleteTissue(): void {
-    this.tissue.deleted = true;
+    if (this.valueChanged('1', 'deleted')){
```

Comment:
the deleteTissue was not calling the value changed and was not communicating with backend